src/bluefinClient.ts

```diff
@@ -530,6 +530,21 @@ export class BluefinClient {
     }`;
   };
 
+  /**
+   * @description
+   * Gets the payload containing key and mesasge to sign
+   * @returns SigPK
+   * */
+
+  signAndCreateZkSignature = async (payload: object): Promise<SigPK> => {
```

Comment:
Hey @izeeshan100, you can please remame this as "signPersonalMessageUsingZkWallet" or "signPayloadUsingZkWallet".

Reply:
@ahsankhan201010  updated. you may check now